test_programs/plonky2_prove_success/less_than_4/Prover.toml

```diff
@@ -0,0 +1,20 @@
+x = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15]
+y = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15]
```

Comment:
These are good, could we just have some larger numbers as well?

So maybe [0, 1, 2, 13, 457, ..., 1000000001], perhaps use an RNG.

Reply:
Ok, I added some more bit patterns (with larger numbers) to the test: https://github.com/blocksense-network/noir/pull/60/commits/925d15d6e9475e9ce59aa3c8889d5214e38133e2

Reply:
Looks great! Merging.